stories/Erase Your Social

```diff
@@ -0,0 +1,16 @@
+To be apart of "Gen Z" comes with privilege and responsibility. With modern medicine, technology, one could argue we live
+in the best of times. The "youth", our young striving adults are often pointed at when the future is mentioned. We must
+continue the cycle to have a better tommorrow, so that the next generation has it that much easier. Or at the very least
+has the oppurtunity to do so. With it being "Gen Z's" turn how well are we stacking up to our predecessors ? I'm sure 
+we've all encountered the stubborn oldhead who swears everything was better in his/her day. We painfully listen and 
+roll our eyes but how much truth is there to be heard ? The biggest change to factor in, (my ever so humble opinion), lies within what 
+we carry in our pockets everyday. Social media has essentially granted us unlimited access to information that is essentially
+catered to us. We see what we wanna see, we hear what we wanna hear, our feed shows us only what we deem as truth. For 
+some, having this unlimated access is more than dangerous. Combine that with how young most of us had this access, we 
```

Comment:
> is more than dangerous
Oh, well do tell then, how dangerous be it my good man? 😄 
which word shall we use to describe the step up the gradient?
- alarming
- critical
- fatal
- perilous

Thoughts? Not hung up on these words.


Minor typo; with a word shift and a timing adjustment.
```suggestion
some, having this unlimited access could be perilous. Combine that with how young most of us were when had this access, we
```

Reply:
I'm not in love with the "could be perilous" part, however, As a reader, I'm curious to get a finer picture into your perspective here.